### PR TITLE
German translation of app strings

### PR DESCRIPTION
**Pull request by peat80**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -1 +1,139 @@
+<resources>
+    <!-- General -->
+    <string name="app_name">Aerial Views</string>
+    <string name="loading_title">Wird geladen...</string>
 
+    <!-- Categories -->
+    <string name="category_capabilities">TV-F�higkeiten</string>
```
Comment (punker76) [inline app/src/main/res/values-de/strings.xml:7]: maybe better: `TV-Funktionen`

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -1 +1,139 @@
...
+    <string name="category_video_sources">Video-Quellen</string>
+    <string name="category_video_source">Video-Quelle</string>
+    <string name="category_actions">Aktionen</string>
+    <string name="category_playlist">Wiedergabeliste</string>
+    <string name="category_network_settings">Netzwerkeinstellungen</string>
+    <string name="category_local_storage_settings">Lokale Speichereinstellungen</string>
+    <string name="category_performance_debug">Performance &amp; Debugging</string>
+
+    <!-- Apple Video Options -->
+    <string name="apple_videos_title">Videos von Apple TV</string>
```
Comment (punker76) [inline app/src/main/res/values-de/strings.xml:18]: I would leave this as `Apple TV Videos`

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -1 +1,139 @@
...
+    <string name="apple_videos_enabled">Nutze Apple Videos</string>
+
+    <string name="apple_videos_quality_title">Qualit�t</string>
+    <string name="apple_videos_quality_summary">Videoqualit�t ausw�hlen</string>
+
+    <string name="apple_videos_data_usage_title">Datenverbrauch</string>
+
+    <!-- Any Video Options -->
+    <string name="any_videos_title">Alle Videos</string>
+    <string name="any_videos_summary">Verwende alle verf�gbaren Videos (Interner Speicher, USB, Netzwerk)</string>
```
Comment (punker76) [inline app/src/main/res/values-de/strings.xml:30]: maybe better `Verwende alle verfügbaren Videos vom Gerätespeicher, USB oder lokalem Netzwerk`

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -1 +1,139 @@
...
+    <string name="interface_summary">Konfiguriere das Aussehen</string>
+
+    <string name="interface_show_clock_title">Zeige Uhr</string>
+    <string name="interface_show_clock_summary">Zeigt die aktuelle Zeit</string>
+
+    <string name="interface_show_location_title">Zeige Videotitel</string>
+    <string name="interface_show_location_summary">Zeigt einen Titel oder Standort des laufenden Videos an</string>
+
+    <string name="interface_show_location_style_title">Apple Videos</string>
+    <string name="interface_show_location_style_summary">W�hle Stil des Videotitels</string>
```
Comment (punker76) [inline app/src/main/res/values-de/strings.xml:96]: `Wähle den Stil des Videotitels`

**Comment by peat80** [comment]
@punker76 The other strings are okay for you? I'm fine with your suggestions also. But let's see what @theothernt thinks about the translations first. 🤔

**Comment by theothernt** [comment]
It's fine with me @peat80. @punker76 is a friend who I asked to take a look.

Also, i've tested the strings in the emulator and they look fine to me?

**Comment by peat80** [comment]
So should i change some wording to the suggestions and then it's good to go? I tend to keep strings as short as possible when translating cause german somtimes has soo long wordings that the translations do not fit the screen. Should probably not be a problem for tv apps cause there is plenty of space available in landscape mode. 😀

**Comment by theothernt** [comment]
Yes please!

Then I'll do one final test on a TV to double check all the text, then merge.

**Comment by peat80** [comment]
Okay @theothernt i have updated the strings as suggested. 😎

**Comment by theothernt** [comment]
One more issue to fix. It seems some of the German characters are not encoded properly eg. `Qualit�t`

I only see a ? (question mark symbol) for those characters on the Android 12 emulator.

**Comment by peat80** [comment]
I think those special german characters (ä ö ü) got broken with your unescaped character fix. I think this has something to do with text edits. Must be saved in UTF-8 i think, otherwise those special characters get broken. 🤔

**Comment by peat80** [comment]
@theothernt Please try again. I think one definetly has to make shure to save file with utf-8 when editing text containing special umlauts.
Replacing umlauts with those codes maybe also works when not saved with utf-8 support. 🤷🏻‍♂️

Ä      &#196;
Ö      &#214;
Ü      &#220;
ä      &#228
ö      &#246;
ü      &#252;
ß      &#223;

Found those on the net and have not tried.

**Comment by theothernt** [comment]
That all looks great to me!

If you don't mind, I just want to change 1 English string - if you could re-translate that then we're good to go?

**Comment by peat80** [comment]
Sounds  like a good plan. 😀👌

**Comment by theothernt** [comment]
> Sounds like a good plan. 😀👌

Ok, small change made 👍🏻

**Comment by peat80** [comment]
Should be ready now. Have a look again. 😀

**Comment by theothernt** [comment]
Thank you so much for translating all that text. Is it ok to mention your Github username in the release notes?

**Comment by peat80** [comment]
Sure, and if there are new stings to translate in the future just give a note. 😁

**Pull request merged**